client/src/components/CreatePetForm.jsx

```diff
@@ -19,21 +18,21 @@ import {
 } from "@material-tailwind/react";
 import { OWNERNAMES, CATBREEDS, DOGBREEDS } from "../utilities/dummydata";
 
-export function CreatePetForm() {
+export function CreatePetForm({ baseURL }) {
   const [species, setSpecies] = useState(null);
-  const [catBreedList, setCatBreedList] = useState([]);
-  const [dogBreedList, setDogBreedList] = useState([]);
+  const [catBreedListOptions, setCatBreedListOptions] = useState([]);
+  const [dogBreedListOptions, setDogBreedListOptions] = useState([]);
```

Comment:
Can you have just one `breedListOptions` ?

Reply:
I have had a massive amount of trouble trying to get breed options to load in correctly without deselecting every time a new value has been selected. There was definitely at one point a version where I just had one breedListOptions; honestly, this has gone through so many iterations that I've forgotten all the ways I've tried it.

I'm currently using useEffect to load in cat and dog breeds when the page is rendered; if my understanding of useEffect is correct, I would be resending those fetch requests every time the breed changed if I used it as a dependency of useEffect. I have honestly been wondering if there is any way to in essence call useEffect twice, because I had also been trying to use it just to watch for changes to species so I could reset breed instead of the convoluted ternary rendering I'm doing now.

Reply:
Added this to issues